samples/bluetooth/nrf_dm/README.rst

```diff
@@ -78,6 +78,22 @@ The addresses of these devices are stored in a list.
 When a device does not respond within a certain period of time, it is removed from the list.
 Also the last measurement results with the peers are stored in this list.
 
+Energy consumption and the number of measurements performed in a given time depend on:
+
+   * Advertiser settings.
+   * Scanner settings.
+   * Additional size of measurement window by ``extra_window_time_us`` in the :c:struct:`dm_request` structure.
```

Comment:
How about "Extended size" instead of "Additional size"?

Reply:
Sounds good.